js/sketchfab.js

```diff
@@ -119,7 +120,13 @@ SketchfabPlugin.prototype.onSubmit = function onSubmit() {
 	var modelId = url.substring(url.lastIndexOf('/') + 1);
 	var text = this.renderText(modelId);
 
-	this.insertAtCursor(text);
+	if ((this.mode || '').toLowerCase() === 'wysiwyg') {
```

Comment:
I don't think I have access to the same kind of editor instance as you do. The textarea element doesn't have the 'composer' prop when i retrieve it from the DOM. Or am I looking at the wrong element ?